<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MSI" failures="3" tests="40" name="com.TestCases.LoginTestCase" time="147.650" errors="0" timestamp="2024-12-26T18:40:14 IST" skipped="0">
  <testcase classname="com.TestCases.LoginTestCase" name="CheckHeading1" time="4.468"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckHeading2" time="2.652"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckHeading3" time="3.682"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckInvalidLogin" time="4.228"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckLogin" time="3.038"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckMacbook" time="3.101"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="AddtoCart" time="2.789"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBox" time="2.765"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckContact" time="4.317"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckWishList" time="2.914"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckPhones" time="2.899"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckTablet" time="3.058"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckCamera" time="3.400"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckMP3" time="4.138"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckLaptopsNotebooks" time="4.216"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckComponents" time="3.095"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckComponents2" time="4.081"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckAboutUs" time="3.917"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckContactUs" time="3.034"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSiteMap" time="3.638"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckAllBrands" time="3.233"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckwishList" time="3.443">
    <failure message="expected [My Wish List] but found [Account Login]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [My Wish List] but found [Account Login]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.TestCases.LoginTestCase.CheckwishList(LoginTestCase.java:355)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- CheckwishList -->
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckNewsletter" time="4.092">
    <failure message="expected [Newsletter Subscribtion] but found [Account Login]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Newsletter Subscribtion] but found [Account Login]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.TestCases.LoginTestCase.CheckNewsletter(LoginTestCase.java:370)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- CheckNewsletter -->
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSpecialOffers" time="3.715"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckTermsConditions" time="4.733"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckHeading4" time="2.917"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckFooter1" time="3.363"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckFooter2" time="3.549"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckFooter3" time="3.817"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckFooter4" time="3.233"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckFooter5" time="3.333"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun1" time="3.911"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun2" time="3.788"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun4" time="3.562"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun5" time="3.614"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun6" time="3.648"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun7" time="4.585">
    <failure message="expected [Hewlett-Packard] but found [Search - HP]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Hewlett-Packard] but found [Search - HP]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.TestCases.LoginTestCase.CheckSearchBoxFun7(LoginTestCase.java:583)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- CheckSearchBoxFun7 -->
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun8" time="5.599"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun9" time="6.074"/>
  <system-out/>
  <testcase classname="com.TestCases.LoginTestCase" name="CheckSearchBoxFun10" time="4.011"/>
  <system-out/>
</testsuite> <!-- com.TestCases.LoginTestCase -->
